fv3gfs/physics/stencils/update_dwind_phys_standalone.py

```diff
@@ -0,0 +1,693 @@
+#!/usr/bin/env python3
+
+import os
```

Comment:
you don't have to necessarily preserve this code, though totally fine if you want to for reference

Reply:
Standalone deleted